glue/sales_val_flagging.py

```diff
@@ -152,6 +149,63 @@ def group_size_adjustment(df, stat_groups: list, min_threshold, condos: bool):
         df: dataframe with newly manually adjusted outlier values.
 
     """
+
+    # Assign sv_outlier_reasons
+    df["sv_outlier_reason1"] = "Null"
+    df["sv_outlier_reason2"] = "Null"
+    df["sv_outlier_reason3"] = "Null"
```

Comment:
**[Question, non-blocking]** Why set these to the string `"Null"` instead of `np.nan`?

Reply:
For some reason I thought it would be more intuitive for analysts. I switched it to nan as that probably plays better with the import / our data lake.